content/doc/book/managing/system-properties.adoc

```diff
@@ -55,406 +55,487 @@ dd {
 </style>
 ++++
 
+[[hudson_ClassicPluginStrategy_useAntClassLoader]]
```

Comment:
I can't seem to see any anchors showing up if I hover over the property, am I missing something?

http://localhost:4242/doc/book/managing/system-properties/

Reply:
That's right, for now it's just an anchor so that you can refer/link to it. For example with the anchors added you can for example go/link to http://localhost:4242/doc/book/managing/system-properties/#hudson_ClassicPluginStrategy_useAntClassLoader directly

But it makes sense to be able to see the link so that you can copy it etc. I can make it a self-link like what `:sectlinks:`does for section titles, but then the style will be that of a link (see screenshots) or we need to tinker with the styles. What do you think?

<img width="833" alt="Screenshot 2020-05-28 at 10 23 19" src="https://user-images.githubusercontent.com/6340362/83117430-58656000-a0cd-11ea-8f42-c1417d596006.png">

Reply:
I think the styling is fine, good to make it clear

Reply:
Alrighty. I'll make the changes.

Reply:
Changes done. (Long live regex!)